ui/collapsible-panel.info/sample/main.reel/main.html

```diff
@@ -0,0 +1,271 @@
+                        cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in
+                        culpa qui officia deserunt mollit anim id est laborum.</p>
+                    <input placeholder="A native HTML input">
+                </div>
+            </div>
+        </div>
+        <h3 style="padding-left: 10px">Example 2: CollapsiblePanel + data-driven repetition + isMultipleSelectionAllowed =
+            false</h3>
+        <div style="width:500px; margin: 15px">
+            <div data-montage-id="example-2-repetition">
+                <div data-montage-id="example-2-collapsiblePanel" class="Main-CollapsiblePanel">
+                    <div data-montage-id="example-2-header" data-arg="header"></div>
+                    <div data-montage-id="example-2-content" data-arg="body"></div>
+                </div>
+            </div>
+        </div>
+        <h3 style="padding-left: 10px">Example 3: CollapsiblePanel + template-driven content + isMultipleSelectionAllowed =
+            true</h3>
+        <div style="width:500px; margin: 15px">
+            <div data-montage-id="example-3-collapsiblePanel-1" class="Main-CollapsiblePanel">
+                <div data-montage-id="example-3-header-1" data-arg="header"></div>
+                <div data-montage-id="example-3-content1" data-arg="body"></div>
+            </div>
+            <div data-montage-id="example-3-collapsiblePanel-2" class="Main-CollapsiblePanel">
+                <div data-montage-id="example-3-header-2" data-arg="header"></div>
+                <div data-arg="body">
+                    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore
+                        et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+                        aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
+                        cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in
+                        culpa qui officia deserunt mollit anim id est laborum.</p>
+                    <input placeholder="A native HTML input">
+                </div>
+            </div>
+        </div>
+        <h3 style="padding-left: 10px">Example 4: CollapsiblePanel + data-driven repetition + isMultipleSelectionAllowed =
+            true</h3>
+        <div style="width:500px; margin: 15px">
+            <div data-montage-id="example-4-repetition">
+                <div data-montage-id="example-4-collapsiblePanel" class="Main-CollapsiblePanel">
+                    <div data-montage-id="example-4-header" data-arg="header"></div>
+                    <div data-montage-id="example-4-content" data-arg="body"></div>
+                </div>
+            </div>
+        </div>
+    </div>
+</body>
+</html>
```

Comment:
Clicking & dragging or double clicking in the header of each of these examples selects text. This behavior is strange, would never appear in a (good) shipping application, and should be fixed in this example.